lib/fastMerge.js

```diff
@@ -0,0 +1,99 @@
+// Mostly copied from  https://medium.com/@lubaka.a/how-to-remove-lodash-performance-improvement-b306669ad0e1
+
+/** // Mostly copied from  https://medium.com/@lubaka.a/how-to-remove-lodash-performance-improvement-b306669ad0e1
```

Comment:
Lets remove this comment here as it is already on the first line of the file

```suggestion
/**
```